### ISSUE TITLE
What the hell is Point2f0 or RGBAf0?

### ISSUE THREAD

**Issue by Masacroso** [MakieOrg/Makie.jl#1437]
Im searching some documentation about these two commands that appear many times in various places of julia code that uses Makie, but I didnt find anything in the documentation of Makie (new and old) or in the internet. Can someone gracefully give me a link to a description or explain what are these two commands? Thank you in advance.

**Comment by jkrumbiegel** [comment]
These are aliases for Float32 element versions of Point and RGBA. But I think we have switched to new aliases by now, Point2f and RGBAf

**Comment by kyhps2** [comment]
Still no documentation about Point2f0, RGBAf0 (Feb, 2023)

**Comment by asinghvi17** [comment]
Yeah, we should probably have a from-scratch tutorial in the docs for this.  I've been guiding some people on how to use Makie so will try to write something up.

**Comment by George9000** [comment]
For others who may arrive here, best I could find: 

- Take a look at the source of Meshes.jl, [file points.jl](https://github.com/JuliaGeometry/Meshes.jl/blob/master/src/points.jl).
- GeometryBasics.jl, file [basic_types.jl](https://github.com/JuliaGeometry/GeometryBasics.jl/blob/master/src/basic_types.jl), AbstractPoint and Simplex documentation
- also the now deprecated GeometryTypes.jl, look at the [documentation](https://juliageometry.github.io/GeometryTypes.jl/latest/types/) on simplex
